web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1713,6 +1713,8 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
 
   @volatile var cometGetTimeout = 140000
 
+  @volatile var putAjaxFnsInContainerSession = false
```

Comment:
Should we do something like this...

```  def addToPackages(what: String) {
    if (doneBoot) throw new IllegalStateException("Cannot modify after boot.");
    otherPackages = what :: otherPackages
  }
```

to prevent modification after `Boot`? This doesn't seem like something that should change mid-application.